### ISSUE TITLE
Center or Expand unique child

### ISSUE THREAD

**Issue by oscar-GP** [nyakaz73/Flutter-Footer#2]
Hi.

Is possible center or expand my unique widget inside footerView? 

My widget is a column with three widgets (icon, divider and elevetedButton).

Thanks.

![Simulator Screen Shot - iPhone SE (2nd generation) - 2021-03-05 at 18 05 18](https://user-images.githubusercontent.com/74998743/110187258-5ad3b380-7ddd-11eb-9ba2-3f5ff628f436.png)

**Comment by nyakaz73** [comment]
Yes its very much possible . Inside the children[] of the FooterView you can have any widget of your choice, Columns ,Center, Containers, Expanded, ListViews, etc. I hope this is helpful :)

**Comment by oscar-GP** [comment]
ok but my widget inside children is a column, I try to use MainAxisSize.max but nothing changes.
